keyboards/foxlab/time_re/universal/info.json

```
@@ -0,0 +1,103 @@
+{
+    "keyboard_name": "Time Re Universal",
```

I missed this on the previous review.

```suggestion
    "keyboard_name": "Time RE Universal",
```

---

@noroadsleft  modified. Thanks